conf/st2.conf

```diff
@@ -27,6 +27,9 @@ enable = False
 mode = proxy
 logging = st2auth/conf/logging.conf
 
+# Base URL to the API endpoint excluding the version (e.g. http://myhost.net:9101/)
+api_url =
```

Comment:
Sure. Even so using localhost does not preclude that possibility. If you think about 1st user experience then running st2 + st2web on the same host is reasonable.